src/components/body/Product.js

```diff
@@ -132,6 +83,68 @@ function checkout(e) {
           <button className="product-add-to-cart-dropdown-color" onClick={(e)=> checkout(e)}>Buy now</button>
         </div>
       </div>
+  )
+}
+
+function ProductTitle({displayProduct}) {
+  return (<div className="product-product-info-header">
+    <h1 className='product-product-title'>
+      {displayProduct.name}
+    </h1>
+  </div>)
+}
+
+function ProductPrice({displayProduct}) {
+  return (<div className="product-price-container">
+    <span className="product-product-subtotal product-product-subtotal-margin"> <strong>${displayProduct.price}</strong></span>
+    <div className="empty-div-rename"></div>
+  </div>)
+}
+
+function ProductDescription({displayProduct}) {
+  return (<div>
+      <h1 className="product-product-subtotal product-product-subtotal-font ">About this item</h1>
+      <li>{displayProduct.description}</li>
+    </div>)
+}
+
+function ProductImage({displayProduct}) {
+  return (
+      <img className='product-product-image' src={displayProduct.img && require(`../../assets/${displayProduct.img}`)}
+           alt={`${displayProduct.name}`}/>)
+}
+
+function ProductInfo({displayProduct}) {
```

Comment:
There are some Product related subcomponents that may be universal enough to extract into a separate module for multiple usage. For example, on the cart page: [This component](https://github.com/SI-Encoding/amazon-clone-flask/blob/bf8fabecca115376514f523c7ac05ff686498a4e/src/components/body/Cart.js#L16)

Reply:
That's a good point. I'll make a new issue for checking for cases like this.